src/utils/csv_generator.py

```diff
@@ -0,0 +1,48 @@
+import random
+import names
+import string
+from faker import Faker
+
+fake = Faker('en_CA')
+domains = ["hotmail.com", "gmail.com", "mail.com" , "mail.ru", "yahoo.com"]
+trades = ["Boilermaker", "Carpenter", "Cement & Concrete Finisher" , "Construction Office Manager", "Crane Operator"]
+letters = string.ascii_letters
+
+def yob_gen(a, b):
+    for x in range(1):
+        v = random.randint(a, b)
+        return v
+
+def name_gen():
+    z = names.get_first_name() + "," + names.get_last_name()
+    return z
+
+def create_csv():
+    f = open('C:\\Users\\dkhad\\Desktop\\generated_data.csv', 'w')
+    f.write(fin_res)
+    f.close
```

Comment:
1. use 
with open() as f;
.....

2. don't use hardcoded file_path, generate it. Use os lib, create /tmp directory in project and store files there
    current_dir = os.path.dirname(__file__)
    queries_path = os.path.join(current_dir, 'queries.txt')